docs/source/whats-new.md

```diff
@@ -1,47 +1,94 @@
 What's new
 ==========
 
-See `GitHub releases page <https://github.com/OSOceanAcoustics/echopype/releases>`_ for the complete history.
+See [GitHub releases page](https://github.com/OSOceanAcoustics/echopype/releases) for the complete history.
 
-v0.5.6 (2022 Feb 10)
---------------------
+# v0.6.0 (2022 May 4)
 
-Overview
-~~~~~~~~
+## Overview
+This is a major release that contains changes that enhances the compliance of echopype data model (and hence generated file structure) to the [SONAR-netCDF4 ver.1 convention](http://www.ices.dk/sites/pub/Publication%20Reports/Cooperative%20Research%20Report%20(CRR)/CRR341.pdf) [LINK REQUIRES LOGIN??? CHECK AGAIN BEFORE RELEASE]. In addition, some variables were renamed to improve intuitive understanding of sonar data, provenance and standardized attributes are added to the processed dataset (e.g. Sv), the deprecated old API (<0.5.0) was removed, and some bugs were fixed.
+
+## Changes of netCDF data model
+
+- Move and rename the original `Beam` and `Beam_power` group to be subgroups under the `Sonar` group, in order to comply with the structure defined in the convention  (#567, #574, #605, #606, #611)
+  - `Beam` --> `Sonar/Beam_group1`: contains either raw power or power/angle data (for all sonar models other than EK80) or complex data (EK80)
+  - `Beam_power` --> `Sonar/Beam_group2`: contains power/angle when complex data occupies `Sonar/Beam_group1`; only exists for EK80 data when both power/angle data and complex data exist in the file
+- Rename the coordinate `range_bin` to `range_sample` to make it obvious that this coordinate indicates the digitization sample number for the sound waveform or intensity time series, and hence it takes a form of sequential integers 0, 1, 2, 3, ... (#595)
+- Rename the data variable `range` in the calibrated Sv or TS dataset to `echo_range`, so that it is not confused with the python reserved vocabulary (#590)
+- Rename the coordinate `quadrant` for EK80 data to `beam` (#619)
+- Add coordinate `beam` with length 1 for all sonar models, except for AD2CP (#638, #646)
+- Rename the data variable `Sp` to `TS` since "point backscattering strength" is a Simrad terminology and target strength (TS) is clearly defined and widely used. (#615)
+- Rename time dimensions in the `Platform` group (`location_time`: `time1`, `mru_time`: `time2`) (#518, #631, #647)
+- Rename the coordinate `frequency` to `channel` for all groups, to be more flexible (can accommodate channels with identical frequencies #490) and reasonable (since for broadband data the channel frequency is only nominal #566) (ADD WHEN KNOWN)
```

Comment:
There's still a `(ADD WHEN KNOWN)` here. Maybe @b-reyes can list out the PR numbers related to this work?